src/App.js

```diff
@@ -1,22 +1,57 @@
-import { FormSetDates } from "./components/Form/FormSetDates";
-
+import {FormInput} from './components/activity/FormInput';
+import {ActivityList} from './components/activity/ActivityList';
+import {useState} from 'react';
+import styled from 'styled-components';
+import {ThemeProvider} from 'styled-components';
+import {StyleHeader} from './components/activity/styles/Header.styles';
 
 
+const theme = {
+  colors: {
+    header: 'white',
+    body: 'rose',
+    footer: 'lightgrey',
+  },
+};
 
 export default function App() {
-  return (
-    <>
-    <heading>
-      <h1>Struct your Day</h1>
-    </heading>
-    
-    
-    
+  const [activities, setActivity] = useState([]);
+
+  const addActivity = activity => {
+    setActivity([...activities, activity]);
+  };
+  const removeActivity = activity => {
+    console.log(activity);
```

Comment:
Please delete this console log. There should be no console logs in your final PR.